composites/Plugin/Shared/components/YoastInput.js

```diff
@@ -18,7 +19,11 @@ export const YoastInputLabel = styled.label`
 `;
 
 export const YoastInputField = styled.input`
-	border: 1px solid ${ colors.$color_input_border };
 	padding: 0.75em;
 	font-size: 1em;
+	&&& {
+		box-shadow: inset 0 1px 2px ${ rgba( colors.$color_black, 0.07 ) };
+		border: 1px solid ${ colors.$color_input_border };
+		border-radius: 0;
```

Comment:
Noticed that setting `box-shadow` and `border` to a so high specificity prevents the focus style to work properly in the metabox. Instead in the sidebar the focus style is inherited by Gutenberg and it works.